### DISCUSSION TITLE
Alternative for https://github.com/KanarekApp/flutter_map_fast_markers

### DISCUSSION THREAD

**Discussion by LamaAgent** [fleaflet/flutter_map#1498] [Q&A]
Hello everyone, is there an alternative to the above plugin that is no longer being developed to load many markers efficiently?

**Comment by JaffaKetchup** [comment]
Hi @LamaAgent,
Apologies for not realising this question was here sooner! We tend to answer more questions on Discord than here.
If it's still useful, there aren't really direct alternatives that I know about. Some of the clustering plugins promise to be really efficient. Maybe a little bit more explanation of what you are trying to do would help?
Once again, apologies for the terrible response time!
